src/index.html

```diff
@@ -1,12 +1,415 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="is-active nav__link">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog">
+        <div class="catalog__card" data-qa="card">
+          <a class="catalog__image" href="/">
+            <img
+                class="catalog__imac"
+                src="images/imac.jpeg"
+                alt="imac"
+              >
```

Comment:
```suggestion
            <img
              class="catalog__imac"
              src="images/imac.jpeg"
              alt="imac"
            >
```
You have too big indents.